ehcache-impl/src/main/java/org/ehcache/impl/internal/store/basic/NopStore.java

```diff
@@ -67,6 +67,11 @@ public void setInvalidationValve(InvalidationValve valve) {
 
   }
 
+  @Override
+  public Iterable<? extends Map.Entry<? extends K, ? extends ValueHolder<V>>> bulkComputeIfAbsentAndFault(Iterable<? extends K> keyParam, Function<Iterable<? extends K>, Iterable<? extends Map.Entry<? extends K, ? extends V>>> mappingFunction) throws StoreAccessException {
+    return null;
```

Comment:
We should implement this in the same manner as `bulkComputeIfAbsent`... it should return a map with null values.

Reply:
incorporated